integration/openshift/e2e/runner.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -e
+
+GOPATH=${GOPATH:-$(go env GOPATH)}
+script_dir="$(realpath $(dirname $0))"
+source "$script_dir/../../../lib/common.bash"
+tests_dir="$script_dir/../../../"
+source "$script_dir/../lib.sh"
+
+cmd="openshift-tests"
+readonly test_cases_file="${script_dir}/test_cases.txt"
+readonly default_kata_runtimeclass="$kata_runtimeclass"
+
+# Whether to run openshift-tests on dry-run mode or not.
+dry_run="false"
+# Whether to configure the OpenShift cluster or not.
+configure_cluster="false"
+
+function usage() {
+	cat << EOF
+This script runs a sub-set of the OpenShift e2e tests in other to test Kata
+Containers in a running OpenShift cluster.
+
+It assumes that the KUBECONFIG variable is exported and that it points to a
+cluster where Kata Containers is already installed.
+
+The cluster also should have:
+ 1) The $default_kata_runtimeclass runtimeClass resource;
+ 2) A running admission controller which annotates any created Pod
+    to use that runtimeClass.
+
+If you choose to have the cluster configured by this script, beware that it
+will not return to its original state on the end.
+
+Usage $0: [-c] [-d] [-r NAME] -o DIRECTORY, where:
+  -c           Configure the cluster to run the tests.
+  -d           Run the tests in dry-run mode.
+  -h           Print this message.
+  -o           Directory to store the output files.
+  -r           The runtimeClass name (default is '$default_kata_runtimeclass').
+               This option should not be used with -c.
+EOF
+}
+
+function setup()
+{
+	mkdir -p $output_dir
+
+	# If the cluster is already configured by the user then only check that the
+	# admission controller is active.
+	#
+	if [ "$configure_cluster" == "true" ]; then
+		[ "$default_kata_runtimeclass" == "$kata_runtimeclass" ] || \
+			die "To configure the cluster it should use the default runtimeClass $default_kata_runtime_class, so do not use the -c and -r options together"
```

Comment:
Nit: super long line ;)

Reply:
I tried to improve, let me know what you think.